frontend/integration-tests/views/crd.view.ts

```diff
@@ -0,0 +1,16 @@
+import { $, browser, by, element, ExpectedConditions as until } from 'protractor';
+
+export const crdItemsList = $('.co-m-resource-customresourcedefinition');
+
+export const clickOneCrd = async function(crdName: string, crdLink: string) {
+  const cs = element(by.linkText(crdName));
+  expect(cs.getAttribute('href')).toContain(crdLink);
```

Comment:
We shouldn't be testing specific `href` values. The view file generally should not have `expect` statements.